chainerx_cc/chainerx/python/routines.cc

```diff
@@ -275,6 +275,17 @@ void InitChainerxCreation(pybind11::module& m) {
           py::arg("endpoint") = true,
           py::arg("dtype") = nullptr,
           py::arg("device") = nullptr);
+    m.def("meshgrid",
+          [](py::sequence arrays, const nonstd::optional<std::string>& indexing) {
```

Comment:
Let's stick to the same names as NumPy where it makes sense, i.e. `xi`.

Reply:
Done.